gcs/src/main/scala/blobstore/gcs/GcsStore.scala

```diff
@@ -123,8 +144,19 @@ object GcsStore {
   def apply[F[_]](
     storage: Storage,
     blocker: Blocker,
-    acls: List[Acl]
-  )(implicit F: Concurrent[F], CS: ContextShift[F]): GcsStore[F] = new GcsStore(storage, blocker, acls)
+    acls: List[Acl] = Nil,
+    defaultTrailingSlashFiles: Boolean = false,
+    defaultDirectDownload: Boolean = false,
+    defaultMaxChunksInFlight: Option[Int] = None
```

Comment:
These needs some scaladoc, in particular `defaultDirectDownload`. I see that the "other" download mechanism has retries of chunks builit in, which probably part of the cause of why it's slower. We should surface this in the docs.